### ISSUE TITLE
IG Automate fetching libraries to include in the docker image

### ISSUE THREAD

**Issue by dbadham-fr** [SecureBankingAccessToolkit/SecureBankingAccessToolkit#1227]
IG dir: `config/7.3.0/securebanking/ig/lib` contains the libraries used by our Groovy scripts and secure-api-gateway-ig-extensions module to include in the docker image.

Currently, third party libs are checked into the git repo.

Our extensions module is included by doing a maven build but there is nothing to check that it is included in the dir when the docker image gets built (this is particularly a problem when building locally). In future, if this module has dependencies of its own then these need to be manually added to the libs dir also.

Improve how the libs directory and docker image get built so that it is automated, this needs to work when run locally and via the GitHub actions.

**Comment by dbadham-fr** [comment]
The current process is cumbersome, see PR to update the bouncy castle dependency:
https://github.com/SecureApiGateway/secure-api-gateway-ob-uk/pull/482

It requires that jars are manually fetched and updated in the /lib dir and, in some cases, the pom updating too. Care needs to be taken to ensure that the contents of /lib match with the pom.

Also, if the dependency has any of its own dependencies, then these need to manually be fetched and added too. When upgrading bouncy castle, we ran into some issues as they have introduced a new dependency on `bcutil-jdk18on` which we did not pull in and caused failures at runtime. PR to fix this issue: https://github.com/SecureApiGateway/secure-api-gateway-ob-uk/pull/483
